### DISCUSSION TITLE
Play multiple Nuvies sequence from a playlist [is it possible?]

### DISCUSSION THREAD

**Discussion by giobbino** [frntc/RAD#53] [Ideas]
I was wondering if there's a way, modifying the RAD firmware, to execute all the nuvies in a playlist (or folder).  I mean a function that automatically: 

- mounts the first nuvie from the playlist or folder;
- plays it once (no loop and of course no initial message at the bottom of the page, just plays immediately);
- unmounts it and mounts the next one;
- plays it once...
etc.

this would work fine especially if you create nuvies without the intro screen. Of course it could happens some little delay when the movie is changed (old nuvie unmounted/new nuvie mounted), but I think this is just a minor issue.

you can already do that somehow manually, pressing the menu button while a nuvie is playing, choosing the next .reu and playing it; for that reason I'm supposing this is not hard to make, some kind of macro without to display the menu and the nuvie title.

This would allow to make extended play of hours...  and the same concept would be extended to the koala video, so in this case you could build a whole movie with audio.

**Comment by frntc** [comment]
Hi,

I think the NUVIE header would include all information to stop playing at the right point in time (or tracking the REU accesses). And of course the RAD could control the entire system.

**Comment by giobbino** [comment]
Great! So, would you kindly add this option to the firmware? ;-)  Because unfortunately I have no skill for that :-(

**Comment by giobbino** [comment]
Any way that somebody would develop this interesting (imo) functionality? This would allow a playback, in example, of an entire movie (of course, without audio)....

**Reply by frntc** [reply]
I'd like to focus on a few other things with higher priority, sorry. Streaming an entire movie is also different from playing one NUVIE after another, because the data needs to be streamed from SD-card then (unless you want a few seconds break every minute or so).

**Comment by giobbino** [comment]
yes, I understand you have other priorities, of course. 
However, even a few seconds break would be good for me, I would only remove the initial title with the nuvie file name at the bottom of the screen...  The perfect solution in this case would be displaying the last frame (as still frame) while the RAD load the next file (if possible).
